src/arguments.js

```diff
@@ -15,14 +15,28 @@ const getArgumentObject = (object, context) => {
         shortName: getPropertyName(object.arg, context.regexp.argument.short),
         defaultValue: getDefaultValue(
             object.description, context.regexp.defaultValue),
-        description: object.description.trim(),
+        description: unifyDescription(object.description),
     };
     argument.isFlag = isDefaultBollean(argument.defaultValue)
         && !isPropertyTyped(object.arg);
 
     return argument;
 };
 
+/**
+ * Unify description to one standard
+ * @param {string} description - argument description
+ * @return {string} description - unified description
+ */
+const unifyDescription = (description) => {
+    description = description.trim();
```

Comment:
I think we need to validate input description by `string` and `length` (if description is empty)

Reply:
Also we can to combine `trim` with another function for `description`

Reply:
@svoboda-rabstvo i've add validate. Trim is better to leave separate.